caps/templates/about_data.html

```diff
@@ -64,7 +66,28 @@ <h2 class="mt-5 mb-4" id="csv">CSV files</h2>
             <li><b>scope</b> - Council only means the plan only deals with the council's operations. Whole area means the plan covers non council activities within council boundaries as well.</li>
         </ul>
 
-        <p>All CSV files include the council GSS code, and a three letter council code you can use with the API, to enable matching data across files, and with other sources.</p>
+        <p>There is also a <a href="{{ MEDIA_URL }}data/projects_data.csv">CSV of the emissions reduction projects data</a> with the following data:</p>
+
+        <ul>
+            <li><b>council_name</b></li>
+            <li><b>authority_code</b> - Three letter council code as above</li>
+            <li><b>start_year</b> - reporting start year</li>
```

Comment:
In the list above (where we describe terms like `last_update` and `plan_path`) the descriptions after the dash start with a capital letter. Would be good to standardise, I don’t mind which way. A capital letter after all of the dashes in the list maybe?